File: src/main/java/no/nav/sosialhjelp/soknad/business/db/repositories/sendtsoknad/BatchSendtSoknadRepository.java
package no.nav.sosialhjelp.soknad.business.db.repositories.sendtsoknad;

import java.util.Optional;

public interface BatchSendtSoknadRepository {

    Optional<Long> hentSendtSoknad(String behandlingsId);
    void slettSendtSoknad(Long sendtSoknadId);
}

File: src/main/kotlin/no/nav/sosialhjelp/soknad/db/repositories/sendtsoknad/BatchSendtSoknadRepository.kt
package no.nav.sosialhjelp.soknad.db.repositories.sendtsoknad

interface BatchSendtSoknadRepository {
    fun hentSendtSoknadIdList(behandlingsIdList: List<String>): List<Long>
    fun slettSendtSoknader(sendtSoknadIdList: List<Long>)
}
